develop.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+POSTGRES_ADMIN_LB_URI="postgresql://postgres:postgres@db/listenbrainz"
```

Comment:
By default, we'd want to log in as the `listenbrainz` user not as `postgres`

Reply:
Oh sorry